src/standalone/Vegetation/radiation.jl

```diff
@@ -46,22 +47,93 @@ ClimaLSM.name(model::AbstractRadiationModel) = :radiative_transfer
 ClimaLSM.auxiliary_vars(model::BeerLambertModel) = (:apar, :par)
 ClimaLSM.auxiliary_types(model::BeerLambertModel{FT}) where {FT} = (FT, FT)
 
+"""
+    TwoStreamParameters{FT <: AbstractFloat}
+
+The required parameters for the two-stream radiative transfer model.
+$(DocStringExtensions.FIELDS)
+"""
+struct TwoStreamParameters{FT <: AbstractFloat}
+    "Leaf angle distribution function (unitless)"
+    ld::FT
+    "PAR canopy reflectance (unitless)"
+    ρ_leaf::FT
+    "leaf element transmittance"
+    τ::FT
+    "Soil albedo"
+    a_soil::FT
+    "Clumping index following Braghiere (2021) (unitless)"
+    Ω::FT
+    "Typical wavelength per PAR photon (m)"
+    λ_γ::FT
+    "number of layers to simulate radiative transfer through"
+    n_layers::UInt64
```

Comment:
does this need to match n_layers of plant hydraulics scheme, @braghiere ?

Reply:
any reason why this is a Uint instead of an Int? I just ask because it is a little unintuitive to see the value as
julia> ts_params.n_layers
0x0000000000000014

instead of 20. might throw someone off?

Reply:
Just because a negative layer count doesn't make any sense using 64 bit Uint gives a larger possible range of valid input values than using a signed 64 bit int, and will automatically throw an error if a user tried to give negative layers.

Reply:
ok! thanks. I think it's OK for now that the n_layers here do not match the n_layers of the hydraulics model. I'm opening an issue just to keep track of it so we can revisit it later and make sure, but all good for now.